application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -142,4 +138,27 @@ static ReminderRoute toPrivate(PrivateChannel channel) {
                             + " the original channel you wanted it to be send to)");
         }
     }
+
+    private void attemptRetryReminder(PendingRemindersRecord pendingReminder, Throwable failure) {
+        if (pendingReminder.getFailureAttempts() > MAX_FAILURE_RETRY) {
+            logger
+                .warn("""
+                        Failed to send a reminder with (authorID '{}') skipping it. This can be due to a network issue, \
+                        but also happen if the bot disconnected from the target guild and the \
+                        user has disabled DMs or has been deleted. It's reminderId '{}' content includes '{}'.""",
+                        pendingReminder.getAuthorId(), pendingReminder.getContent(),
+                        pendingReminder.getId(), failure);
+            return;
+
+        }
+
+        int failureAttempts = pendingReminder.getFailureAttempts() + 1;
+        Instant remindAt = Instant.now().plus(1, ChronoUnit.MINUTES);
```

Comment:
y not set it to 1 min after the initial reminding time?

Reply:
Not sure, followed what zabu suggested.

Reply:
i had a feeling that since `.queue()` was used on each api call then it could be delayed a bit as well